user-profile-card.js

```diff
@@ -201,17 +213,23 @@ class UserProfileCard extends LocalizeMixin(LitElement) {
 
 	constructor() {
 		super();
+		this.editable = false;
 		this.online = false;
+		this.tagline = '';
 		this.userAttributes = [];
+		this._isEditing = false;
 	}
 
 	render() {
 		this.userAttributes.map((item) => {
 			console.log(item);
 			return html`<li>${item}</li>`;
 		});
+
+		const classes = { 'd2l-labs-profile-card' : true, 'd2l-is-editing' : this._isEditing};
```

Comment:
Same thing here - keep object spacing consistent. By convention, there is no space between the key and the colon.
```suggestion
		const classes = { 'd2l-labs-profile-card': true, 'd2l-is-editing': this._isEditing};
```